src/NotepadNext/dialogs/MainWindow.ui

```diff
@@ -40,7 +40,7 @@
      <x>0</x>
      <y>0</y>
      <width>855</width>
-     <height>22</height>
+     <height>25</height>
```

Comment:
Was this change intentional? Thought I'm not sure this would even have any affect.

Reply:
It wasn't intentional. Some of the actions are added by the designer and when I change the order and remove one of the separators (it is put back by code), this height changes.